cmd/notary/tuf.go

```diff
@@ -123,6 +125,7 @@ func (t *tufCommander) AddToCommand(cmd *cobra.Command) {
 
 	cmdTUFAdd := cmdTUFAddTemplate.ToCommand(t.tufAdd)
 	cmdTUFAdd.Flags().StringSliceVarP(&t.roles, "roles", "r", nil, "Delegation roles to add this target to")
+	cmdTUFAdd.Flags().BoolVarP(&t.autoPublish, "publish", "p", false, "Automatically attempt to publish after staging the change. Will also publish existing staged changes.")
```

Comment:
I agree with @cyli - also for when we implement the `delegation add` and `delegation remove` auto-publish flags, I think the text will be the same, and we've used constants for similar important text like `tufRootKeyGenerationWarning`